pkg/controller/node/controller_utils.go

```diff
@@ -164,13 +164,13 @@ func (nc *NodeController) maybeDeleteTerminatingPod(obj interface{}) {
 	// guarantee backwards compatibility of master API to kubelets with
 	// versions less than 1.1.0
 	node := nodeObj.(*api.Node)
-	v, err := version.Parse(node.Status.NodeInfo.KubeletVersion)
+	v, err := utilversion.ParseSemantic(node.Status.NodeInfo.KubeletVersion)
 	if err != nil {
 		glog.V(0).Infof("couldn't parse verions %q of minion: %v", node.Status.NodeInfo.KubeletVersion, err)
 		utilruntime.HandleError(nc.forcefullyDeletePod(pod))
 		return
 	}
-	if gracefulDeletionVersion.GT(v) {
+	if !v.AtLeast(gracefulDeletionVersion) {
```

Comment:
I was trying to see if I could get away with only a single comparison operator rather than, eg, having all of EQ, NE, GT, GTE, LT, and LTE like blang/semver. It seemed to me that in most cases, you have some version of the software where a feature was added, and you want to know if the version of the software you're running is at least that version or not.

But maybe having the reverse check as well would be good.